src/net/ssl-context.cpp

```diff
@@ -75,7 +80,6 @@ SslContext::SslContext(std::string ca_bundle_path) {
 }
 
 SslContext::~SslContext() {
-    EVP_cleanup();
```

Comment:
This was actually a bug that caused us to de-initialize OpenSSL globals every time a `SslContext` died